IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcGeographicCRS/DocEntity.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeographicCRS" UniqueId="2d6be663-8d5e-42cc-b5b4-13bb37f41262" BaseDefinition="IfcCoordinateReferenceSystem" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PrimeMeridian" UniqueId="b2c9cf50-b4bb-4979-92d8-c44a05f22954" DefinedType="IfcIdentifier" AttributeFlags="1">
+			<Documentation>The identification of the meridian defining zero longitude in the used geographic CRS.</Documentation>
+			<Definition>
+				<DocDefinitionRef xsi:nil="true" href="IfcIdentifier_1dG8O0DIbDBxufYev6dUDH" />
+			</Definition>
+		</DocAttribute>
+		<DocAttribute Name="Unit" UniqueId="82bf71dc-4f74-483b-84c7-70c84a5ce22d" DefinedType="IfcNamedUnit" AttributeFlags="1" />
+	</Attributes>
+	<WhereRules>
+		<DocWhereRule Name="UnitIsPlaneAngle" UniqueId="45472275-87bf-4f5e-a1ad-80be7f068991">
+			<Documentation>The type of Unit in the operation shall be _IfcPlaneAngleMeasure_.</Documentation>
```

Comment:
I'm confused a bit, when a *unit* shall be *measure*. Perhaps we can change the wording to better reflect that the unit should be a plane angle *unit*? (I'm unsure how this is worded in the rest of the standard.)